tests/Unit/Domain/CoordinateMaps/Test_CoordinateMap.cpp

```diff
@@ -748,6 +758,98 @@ void test_make_vector_coordinate_map_base() {
   CHECK(*translated_lower_xi_wedge_base == *(vector_of_translated_wedges[2]));
   CHECK(*translated_lower_eta_wedge_base == *(vector_of_translated_wedges[3]));
 }
+
+void test_coordinate_maps_are_identity() {
+  CHECK(CoordinateMaps::Affine{-1.0, 1.0, -1.0, 1.0}.is_identity());
```

Comment:
Is there a reason these are all grouped together here instead of being in the tests for the maps themselves?